CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Version 1.9.0
+
+- Patch a weak-increment race condition (#60)
+- Manually implement `Clone` on `WeakSender/WeakReceiver` (#62)
```

Comment:
These seem to be the titles of PRs or commits, and they may explain what they did, but in the changelog, I think writing what has been fixed/changed by the PR would be easier for users to understand.

- In the first case, is that "fix a bug that weak*upgrade could incorrectly return Some even if the channel is already closed"? (And the underlying channel is already closed, so it was always be an error?)
- In the second case, is that "remove unnecessary `T: Clone` bound from weak*'s Clone impl"?

Reply:
Sounds good, thanks!